app/models/user.js

```diff
@@ -34,22 +34,19 @@ export default EmberObject.extend({
     }).then((response) => {
       if (response.ok) {
         return response.json().then(data => data.user_id);
-      } else {
-        if (response.status === 401) {
-          this.logger.info('Token not authorized by Helios');
-        } else {
-          this.logger.error('Helios connection error: ', response);
-        }
-
-        return null;
       }
+      if (response.status === 401) {
+        this.logger.info('Token not authorized by Helios');
+      }
+      this.logger.error('Helios connection error: ', response);
```

Comment:
now this will be logged always
and used to be logged only when response.status !== 401